sdk/core/core-asynciterator-polyfill/package.json

```diff
@@ -2,7 +2,7 @@
   "name": "@azure/core-asynciterator-polyfill",
   "author": "Microsoft Corporation",
   "sdk-type": "client",
-  "version": "1.0.2",
```

Comment:
amusing that this got over-incremented

Reply:
I did some searches around the internet, git history, etc. to make sure that 1.0.1 wasn't floating around in outer space somewhere.